dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -1,7 +1,7 @@
 /*
  *    AutoComplete interfaces implementation.
  *
- *    Copyright 2004    Maxime Belleng� <maxime.bellenge@laposte.net>
+ *    Copyright 2004    Maxime Bellengé <maxime.bellenge@laposte.net>
```

Comment:
If this is a real change, and no fuckup of the gut review webinterface, then please do revert it!
Same in the 2nd file that was touched.

Reply:
Perhaps it's an ANSI -> UTF8 conversion made in the file, in which case I would say it's fine.

Reply:
Why does it look worse than before then, if you switch into githubs "Files changed"-tab?
If it is really just wrong in the webview, then please ignore my rejection.

Reply:
it used to be ansi indeed, now it's utf8